### ISSUE TITLE
Long value wrongly converted and displayed

### ISSUE THREAD

**Issue by riveria** [callumlocke/json-formatter#117]
The plain source:

`"transactionInfo":{"messierTransactionId":4692781588633638289}}`

The number is then converted to a different number and is displayed wrongly through JSON formatter:

{
  transactionInfo: {
  messierTransactionId: 4692781588633638000
  }
}

It's interesting that JSON formatter did that "4692781588633638289" -> "4692781588633638000".

**Comment by Techtony96** [comment]
I am also having this issue, it seems to be cutting off the two last digits and displaying them as zeros.

**Comment by alexwhittemore** [comment]
This appears to be addressed in the readme: https://github.com/callumlocke/json-formatter/blob/master/README.md

**Comment by Techtony96** [comment]
I disagree with that statement, JSON is just a format and not only used by JavaScript. There are plenty of languages that use JSON and can correctly use large numbers.

If this chrome extension was advertised as a JavaScript JSON viewer then maybe I would agree, but seems like the issue is just being avoided.

**Comment by alexwhittemore** [comment]
I tend to agree, but then that's probably why there are 102943832738494 JSON visualizers in the extension store.

**Comment by Techtony96** [comment]
I feel like it would be best if this was the default behavior and there was some way to make it show the actual values as I use JSON from within Java which supports these larger numbers.
